<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>652</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>1081</width>
      <height>561</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="TOTALPATTERN">
     <attribute name="title">
      <string>Over AMISR</string>
     </attribute>
     <widget class="MplWidget" name="MplWidget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>601</width>
        <height>531</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_Altitude">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>160</y>
        <width>121</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>ALTITUDE (km)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_Elevation">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>80</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>ELEVATION</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="fecha_in">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>40</y>
        <width>110</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_azimuth">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>110</y>
        <width>113</width>
        <height>30</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLabel" name="label_Azimuth">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>80</y>
        <width>66</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>AZIMUTH</string>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>230</y>
        <width>301</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addbeam">
         <property name="text">
          <string>ADD BEAM</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_9">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_del">
         <property name="text">
          <string>Delete 1</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_10">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget_2">
      <property name="geometry">
       <rect>
        <x>890</x>
        <y>190</y>
        <width>181</width>
        <height>29</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <spacer name="horizontalSpacer_11">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_plot">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>PLOT</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_12">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QLineEdit" name="lineEdit_high">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>190</y>
        <width>251</width>
        <height>27</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_elevation">
      <property name="geometry">
       <rect>
        <x>800</x>
        <y>110</y>
        <width>181</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget_3">
      <property name="geometry">
       <rect>
        <x>930</x>
        <y>230</y>
        <width>141</width>
        <height>51</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_13">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_refresh">
         <property name="text">
          <string>RESET</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_14">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>48</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QLabel" name="label_date">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>20</y>
        <width>66</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DATE</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_lista">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>300</y>
        <width>401</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="SETEXPERIMENTS">
     <attribute name="title">
      <string>Experiment</string>
     </attribute>
     <widget class="QLabel" name="label_IPP">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>66</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>IPP (usec)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_IPP">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>70</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_Tx">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>nTxA (usec)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>66</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>S Code</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Tx">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nCode">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>230</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nBaud">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>280</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_ndh">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>330</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nsa">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>380</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nProfBlock">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>430</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nBlockFile">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>70</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>66</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>n Code</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>66</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>n Baud</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>330</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ndh (useg)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>380</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>nsa (useg)</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>430</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>n Prof Block</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>70</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>n Block File</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>66</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>S Beam</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>190</y>
        <width>66</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>nFFT</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Sample Start</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SBeam">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>120</y>
        <width>221</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_nFFT">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>190</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_SampleStart">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>240</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_PPS">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>290</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>PPS (seconds)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_titleExp">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>831</width>
        <height>27</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>NAME OF THE EXPERIMENT</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_GEN_EXP">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>420</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>GENERATE EXP.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>340</y>
        <width>71</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>FILES </string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_files">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>340</y>
        <width>221</width>
        <height>51</height>
       </rect>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_SCode">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>141</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>User Name</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_User">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>70</y>
        <width>311</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>120</y>
        <width>391</width>
        <height>271</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_send_exp">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>420</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>SEND EXP.</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_close">
      <property name="geometry">
       <rect>
        <x>810</x>
        <y>484</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>420</y>
        <width>391</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSELECCION_DE_APUNTES_DINAMICOS">
    <property name="title">
     <string>SELECCION DE APUNTES DINAMICOS</string>
    </property>
   </widget>
   <addaction name="menuSELECCION_DE_APUNTES_DINAMICOS"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MplWidget</class>
   <extends>QWidget</extends>
   <header>mplwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../../../Desktop/app/QT_des/amisr_image.qrc"/>
 </resources>
 <connections/>
</ui>
